BookClub/admin.py

```diff
@@ -3,6 +3,7 @@
 from BookClub.models.club import Club
 from BookClub.models.club_membership import ClubMembership
 from BookClub.models.book import Book
+from BookClub.models.review import BookReview
 from BookClub.models.booklist import BookList
 from BookClub.models.meeting import Meeting
 
```

Comment:
Maybe change to just from BookClub.models import *
instead of having to import each model separately